File: week-1/transpose-matrix/transpose-matrix.py
class Solution:
    def transpose(self, A: List[List[int]]) -> List[List[int]]:
        if len(A) == 0:
            return []
        
        res = []
        for j in range(len(A[0])):
            temp = []
            for i in range(len(A)):
                temp.append(A[i][j])
            res.append(temp)
        
        return res

File: week-1/transpose-matrix/transpose-matrix.js
/**
 * @param {number[][]} A
 * @return {number[][]}
 */
var transpose = function(A) {
    if (A.lenth === 0) return [];
    res = []
    for (let j=0; j<A[0].length; j++) {
        temp = [];
        for (let i=0; i<A.length; i++) {
            temp.push(A[i][j]);
        }
        res.push(temp);
    }
    
    return res;
};
